docs/src/routes/docs/components/autocomplete-field.svx

```diff
@@ -120,6 +120,18 @@ The loading state of the dropdown while the next promise with suggestions is res
 The element to display at the end of a list to trigger the option generator.
 Defaults to a button equipped with an `IntersectionObserver` to trigger the generator whenever it scrolls into view.
 
+### `close-message` slot
+
+Use this slot to translate the "close the options" message that appears when tabbing into the dropdown of options.
+
+### `loading-message` slot
+
+Use this slot to translate the "loading" message that appears when the options are loading.
+
+### `load-more-options-message` slot
+
+Use this slot to translate the "load more options" message.
```

Comment:
I was hoping you'd write explanations for all of them, not just for the ones I've left a comment on
```suggestion
Use this slot to translate the "load more options" message that appears when you scroll to the bottom of the list of options before the infinite scroll triggers loading itself.
```